internal/types/host/keepalived_host_collector_host.go

```diff
@@ -30,7 +30,7 @@ type KeepalivedHostCollectorHost struct {
 }
 
 // NewKeepalivedHostCollectorHost is creating new instance of KeepalivedHostCollectorHost
-func NewKeepalivedHostCollectorHost(useJSON bool, pidPath string) *KeepalivedHostCollectorHost {
+func NewKeepalivedHostCollectorHost(useJSON bool, pidPath string) collector.Collector {
```

Comment:
Why should we change the return type to an interface?

Reply:
No need. That was my fault :D